genai-perf/tests/test_genai_perf_config.py

```diff
@@ -62,9 +62,16 @@ def test_parameters(self):
         """
         expected_parameters = {"num_dataset_entries": 50}
 
-        self.assertEqual(
-            expected_parameters, self._default_genai_perf_config.get_parameters()
-        )
+        expected_parameters["endpoint"] = self._config.endpoint.to_json_dict()
+        del expected_parameters["endpoint"]["server_metrics_urls"]
+        del expected_parameters["endpoint"]["url"]
```

Comment:
Remind me. What necessitates this?
I worry this will become the new "skip list" that has to be added to over time.

Reply:
The basic premise is that we shouldn't be storing parameters that have no effect on the metrics being produced, because they get checkpointed.

Thankfully, there are very few fields that this is relevant for and I don't expect it will grow much.

Reply:
I think if this grows much, we need to create a class that is populated from the endpoint config and trims these by default. 
For now, this is fine. Just need to keep an eye on it given that you fixed many of those exact problems for us with this PR.